html/includes/api_functions.inc.php

```diff
@@ -315,6 +315,11 @@ function list_devices()
     }
     $devices = array();
     foreach (dbFetchRows("SELECT * FROM `devices` $join WHERE $sql ORDER by $order", $param) as $device) {
+        $dev_deps = dbFetchColumn("SELECT parent_device_id from device_relationships WHERE child_device_id = ?", array($device));
```

Comment:
I already expect that people will ask for hostnames as well as ID's. Here's a totally unformatted but working example:

`select d.device_id, d.hostname, GROUP_CONCAT(dd.device_id), GROUP_CONCAT(dd.hostname) as parent_hostname from devices as d left join device_relationships as dr on dr.child_device_id=d.device_id left join devices as dd on dr.parent_device_id=dd.device_id GROUP BY d.hostname`